### ISSUE TITLE
Aiming Conversion Wand at Storage Drawers crashes client

### ISSUE THREAD

**Issue by irisfofs** [HellFirePvP/AstralSorcery#1608]
Astral Sorcery version: 1.16.4-1.13.8 
Storage Drawers version: 8.2.1
modpack: [All the Mods 6-1.3.7](https://www.curseforge.com/minecraft/modpacks/all-the-mods-6/)

Aiming a Conversion Wand at storage drawers crashes the client. The crash seems to be related to computing the converted block preview. This is especially insidious because it's likely you will be in the same position when you reconnect, causing another crash before you can do anything and essentially softlocking you out of the server. 

[crash-2020-12-27_23.32.35-client.txt](https://github.com/HellFirePvP/AstralSorcery/files/5752394/crash-2020-12-27_23.32.35-client.txt)
[crash-2020-12-27_23.42.12-client.txt](https://github.com/HellFirePvP/AstralSorcery/files/5752395/crash-2020-12-27_23.42.12-client.txt)

The bug might not actually be in Astral Sorcery, but it was at the very least involved.

**Comment by Jeuvke** [comment]
Same issue as #1418.
TL;DR: it's the fault of Storage Drawers

**Comment by irisfofs** [comment]
Neat. I've +1'd the bug over there: https://github.com/jaquadro/StorageDrawers/issues/902
